_data/resume.yml

```diff
@@ -0,0 +1,48 @@
+fullname: Blanca Azucena López Garduño
+address: Ecatepec 55100, Estado de México, México
+email: blankazucenalg@gmail.com
+linkedin: blankazucenalg
+languages: Spanish, English
+education: Computer Science Engineer, ESCOM, IPN (2011-2016)
+about_me: I fell in love with software development when I discovered that it could be used to solve problems that seemed impossible. I enjoy creating solutions that aim to improve people's lives, share ideas and learn from each experience, always trying to do better. I love music and I don't like horror movies.
```

Comment:
`always trying to do better` Creo que queda mejor `share ideas and learn from each experience, always trying to improve myself`